coast/TIDEGAUGE.py

```diff
@@ -624,51 +614,48 @@ def read_EA_API_to_xarray(
         cls.date_end = date_end
         cls.stationId = stationId  # EA id: stationReference
 
-        #%% Obtain and process header information
+        # %% Obtain and process header information
         info("load station info")
-        url = "https://environment.data.gov.uk/flood-monitoring/id/stations/" + cls.stationId + ".json"
+        url = 'https://environment.data.gov.uk/flood-monitoring/id/stations/' + cls.stationId + '.json'
         try:
             request_raw = requests.get(url)
             header_dict = json.loads(request_raw.content)
         except ValueError:
-            print(f"Failed request for station {cls.stationId}")
+            debug(f"Failed request for station {cls.stationId}")
             return
 
         try:
-            header_dict["site_name"] = header_dict["items"]["label"]
-            header_dict["latitude"] = header_dict["items"]["lat"]
-            header_dict["longitude"] = header_dict["items"]["long"]
+            header_dict['site_name'] = header_dict['items']['label']
+            header_dict['latitude'] = header_dict['items']['lat']
+            header_dict['longitude'] = header_dict['items']['long']
         except:
```

Comment:
We should avoid using bare excepts. These will catch **everything**. Including system exits.